### ISSUE TITLE
SOCKS5 Proxy doesn't seem to work?

### ISSUE THREAD

**Issue by n1nj4888** [morpheus65535/bazarr#1327]
**Describe the bug**
When SOCKS5 proxy is configured, the configured provider returns an error as follows. The provider works with the SOCKS5 proxy config removed.

The same SOCKS5 proxy configuration works in Radarr

**To Reproduce**
1. Go to Settings -> General
2. Configure a SOCKS5 Proxy
3. Search for some subs
4. See Error in System -> Logs:

```
21/03/2021 22:51:00|INFO    |root                            |Provider opensubtitles throttle count 1 of 5, waiting 5sec and trying again|
21/03/2021 22:51:05|INFO    |root                            |Provider opensubtitles throttle count 2 of 5, waiting 5sec and trying again|
21/03/2021 22:51:10|INFO    |root                            |Provider opensubtitles throttle count 3 of 5, waiting 5sec and trying again|
21/03/2021 22:51:16|INFO    |root                            |Provider opensubtitles throttle count 4 of 5, waiting 5sec and trying again|
21/03/2021 22:51:38|INFO    |root                            |Throttling opensubtitles for 20 minutes, until 21/03/21 23:11, because of: ServiceUnavailable. Exception info: 'Unknown Error, empty response: None: None'|

```
**Expected behavior**
Bazarr should use the configured SOCKS5 proxy and, as long as the config is correct, should route through the proxy when communicating outwards.

**Screenshots**
N/A

**Software (please complete the following information):**
 - Bazarr: 0.9.2
 - Radarr: 3.0.2.4552
 - OS: linuxserver/bazarr (Docker)

**Additional context**
N/A

**Comment by morpheus65535** [comment]
That's unrelated to your proxy config. It's just opensubtitles that is unavailable or is throttling your IP. If it's a shared IP, it could be used by other clients than you which could result in temporary IP block.

**Comment by n1nj4888** [comment]
I was also getting the following form of error (which I didn’t post before because the credentials were logged in the clear) - The `<PASSWORD>` previously had a # symbol in it which appears to be the reason it was failing to parse since when I changed the proxy password with the provider and restarted bazarr, it began to work... does this suggest that bazarr is not escaping special characters in the proxy password correctly?

`Throttling opensubtitlescom for 10 minutes, until 21/03/22 04:43, because of: InvalidURL. Exception info: LocationParseError('Failed to parse: socks5://<USER>:<PASSWORD>@<PROXY URL>:<PORT>')
`

**Comment by morpheus65535** [comment]
You're right, the username and password were not encoded and could be the cause fo your issue. Should be fixed in v0.9.3-beta.15. If you're able to test and confirm, I would appreciate. :-)

**Comment by morpheus65535** [comment]
No news, good news. Re-open if required.
